python/cuml/test/dask/test_datasets.py

```diff
@@ -152,5 +155,27 @@ def test_make_regression(n_samples, n_features, n_informative,
 
             assert test2, "Unexpectedly incongruent outputs"
 
+        X_part = c.sync(_extract_partitions, out)
+        out_part = X_part[0][1].result()
+
+        y_part = c.sync(_extract_partitions, values)
```

Comment:
We should also use `DistributedDataHandler` in our tests. I think we should strive to make the change as we encounter them rather than doing it all at once. Before long, we'll be using it everywhere.